objects/9ad187/ui.xml

```diff
@@ -174,6 +174,9 @@
             <Row class="toggle">
                 <Cell class="toggle"><Text class="toggle"/><Toggle id="thematicRedo" onValueChanged="toggleThematicRedo"/><Text class="toggle"/><Button class="doubletoggle" onClick="toggleThematicRedo" preferredWidth="780" tooltip="An alternate appeareance for the Thematic boards,&#xA;so the terrains look like the Balanced boards">Use Recolored Thematic Island Boards</Button></Cell>
             </Row>
+            <Row class="toggle">
```

Comment:
does this not require the height for the panel to be changed?

Reply:
Hmm, the height doesn't appear to have any effect (though removing both height/width makes it disappear).

Reply:
Whoops not the height then, but the XY offset of the UI element below it

Reply:
I've adjusted those.